fitbit-react/src/Util/sort.js

```diff
@@ -0,0 +1,11 @@
+const sortByHighToLow = (products) => {
+  products.sort(
+    (a, b) => parseInt(b.discountedPrice, 10) - parseInt(a.discountedPrice, 10)
+  );
+  return products;
+};
+const sortByLowToHigh = (products) => {
+  products.sort((a, b) => a.discountedPrice - b.discountedPrice);
```

Comment:
Suggestion: make copy of product list or use  spread operator  with `product`, because `sort` mutates array and in `pure functions` we don't mutate even `arguments` .